src/test_l1_dsHost.c

```diff
@@ -273,9 +277,11 @@ void test_l1_dsHost_negative_dsHostTerm(void) {
     UT_LOG("Step 03: Terminate dsHost after initialization -> Expected: dsERR_NONE, Got: %d\n", result);
 
     // Step 04: dsHostTerm() Attempt to terminate dsHost again
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     result = dsHostTerm();
```

Comment:
Comment only the error code, not the API. Also, add dsERR_NONE, which is the expected error code, in the else case.

Reply:
Done